docs/concepts/why-filecoin.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Why Filecoin?
+---
+
+# Why Filecoin?
+
+## Open market
+
+In Filecoin, file storage and retrieval is negotiated in open markets. Anybody can join the Filecoin network without needing permission. Running a miner only requires an internet connection and spare disk space. By lowering the barriers to entry, Filecoin enables a thriving ecosystem of many independent storage providers.
+
+## Competitive prices
+
+Prices for storage and retrieval are determined by supply and demand, not corporate pricing departments. Filecoin makes reliable storage available at hypercompetitive prices. Miners compete based on their storage, reliability and speed rather than through marketing or attempting to lock users in.
+
+## Reliable storage
+
+Because storage is paid for, Filecoin provides a viable economic reason for files to stay available over time. Files are stored on computers that are reliable and well-connected to the internet. Although Filecoin is a peer-to-peer network, it does not rely solely on an ephemeral swarm of peers for files to stay online.
+
+## Self-healing
+
+The Filecoin network continually verifies that files are stored correctly. The Filecoin blockchain has a built-in self-healing process where faulty miners are detected and their files are redistributed to reliable miners.
+
+## Verifiable traces
+
+In the process of self-healing, Filecoin generates verifiable traces that files have been stored correctly over time. Clients can efficiently scan these traces to confirm that their files have been stored correctly, even if the client was offline at the time. Any observer can check any miner’s track record and will notice if the miner has been faulty or offline in the past.
+
+## Reputation not marketing
+
+In Filecoin, storage providers prove their reliability through their track record published on the blockchain, not through marketing claims published by the provider themselves. Users don’t need to rely on status pages or self-reported statistics from storage providers.
+
+## Choice of tradeoffs
+
+Users get to choose their own tradeoffs between cost, redundancy and speed. Users are not limited to a set list of data centres offered by their provider, but can choose to store their files on any miner participating in Filecoin.
+
+## Puts empty storage to use
+
+Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivised to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.
```

Comment:
```suggestion
Filecoin puts disk space that would otherwise sit empty to use. Miners are incentivized to join Filecoin because they get paid for their participation, unlike other distributed networks where there is no economic incentive. Filecoin also makes it easy to join because miners self-manage their storage, earning Filecoin tokens without needing human supervision.
```